test/runtests.py

```diff
@@ -66,6 +68,17 @@ def queue_name(self):
         return self.QUEUE_NAME
 
 
+class SecondaryQueueConsumer(consumers.PulseTestConsumer):
```

Comment:
Mind refactoring to use a base class so we don't have to define queue_name() each time?